.github/workflows/ci-pull.yml

```
@@ -25,7 +24,7 @@ jobs:
     - name: Set up JDK 21 #${{ matrix.java-version }}
```

```suggestion
    - name: Set up JDK ${{ matrix.java-version }}
```
Looks like this got lost in the merge